README.md

```diff
@@ -7,7 +7,7 @@ Package license: Apache-2.0
 
 Feedstock license: [BSD-3-Clause](https://github.com/conda-forge/bentoml-feedstock/blob/main/LICENSE.txt)
 
-Summary: A framework for machine learning model serving
+Summary: Bentoml: The Unified Model Serving Framework
```

Comment:
```suggestion
Summary: BentoML: The Unified Model Serving Framework
```

Reply:
The README is updated automatically every time when the recipe is run. Since the summary has been changed in the recipe, this will take care of the summary in the README file.